toolbar.less

```diff
@@ -40,6 +48,91 @@
             color: @dark-charcoal;
             background-color: @color-list-item-bg-hover;
         }
+
+        .input-group {
```

Comment:
https://github.com/seanCodes/bootstrap-less-port/tree/master/less

Would we be able to import `.input-group` if we used the above library?

I have forked it to https://github.com/hacktoolkit/bootstrap-less-port